src/tests/multihost/sssd/testlib/common/helper_functions.py

```diff
@@ -0,0 +1,47 @@
+"""
+This script contains helper functions that can be used in test.
+"""
+
+import time
+from sssd.testlib.common.expect import pexpect_ssh
+
+ds_instance_name = 'example1'
+
+
+def clear_only_domain_log(multihost):
```

Comment:
we have the method for this,
https://github.com/SSSD/sssd/blob/469905bfabdda66555bd179eb7b6ac7b3cd875d1/src/tests/multihost/sssd/testlib/common/utils.py#L546

can't we use the same?